plugin/kubernetes/handler_test.go

```diff
@@ -643,8 +643,9 @@ var epsIndex = map[string][]*object.Endpoints{
 				},
 			},
 		},
-		Name:      "svc1",
+		Name:      "svc1-aaaaa",
```

Comment:
why is there -`aaaaa` added here?

Reply:
This is what endpointSlice names look like... by convention: the service name, dash, 5 random alpha-nums.
Having this reflected in the tests that we do not have assumptions in the code that and endpoint name = service name, which is no longer the case for EndpointSlice.

Reply:
really, wow? Please add a comment with ref to some docs, this looks like someone was just banging the keyboard, not a conscious design decision.

Reply:
I'll make them just incremental numbers.  The random string is the name scheme that K8s controllers use when creating endpoints (similar to how Pods are named when deployed per Deployments). I was just imitating that to make the data more "real", but it's not necessary at all.  All that matters is that they don't match the service name.